x-pack/plugins/security_solution/server/lib/timeline/routes/README.md

```diff
@@ -323,4 +323,292 @@ kbn-version: 8.0.0
      "timelineId":"f5a4bd10-83cd-11ea-bf78-0547a65f1281", // This is a must as well
      "version":"Wzg2LDFd" // Please provide the existing timeline version
 }
-```
\ No newline at end of file
+```
+
+## Export timeline api
+
+#### POST /api/timeline/_export
+
+##### Authorization
+
+Type: Basic Auth
+
+username: Your Kibana username
+
+password: Your Kibana password
+
+
+
+
+##### Request header
+
+```
+
+Content-Type: application/json
+
+kbn-version: 8.0.0
+
+```
+
+##### Request param
+file_name	${filename}.ndjson
+
+
+##### Request body
+```json
+{
+	ids: [
+		${timelineId}
+	]
+}
+```
+
+## Import timeline api
+
+#### POST /api/timeline/_import
+
+##### Authorization
+
+Type: Basic Auth
+
+username: Your Kibana username
+
+password: Your Kibana password
+
+
+
+
+##### Request header
+
+```
+
+Content-Type: application/json
+
+kbn-version: 8.0.0
+
+```
+
+##### Request body
+a .ndjson file
+(each json in the file should match this [format](https://github.com/elastic/kibana/blob/master/x-pack/plugins/siem/public/graphql/types.ts#L118-L146))
```

Comment:
this link is no good

Reply:
right, we've change the name of the plugin!!

Reply:
I'll remove there link here as we are not going to use graphql, might not be a good idea to refer to that.